Assets/HoloToolkit/Utilities/Scripts/Performance/PerformanceDisplay.cs

```diff
@@ -0,0 +1,92 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public class PerformanceDisplay : MonoBehaviour
+    {
+        public float DisplayInterval = 0.5f;
+        [SerializeField]
+        private KeyCode fpsToggleKey = KeyCode.F;
+
+        UnityEngine.UI.Text fpsGUIText;
+        private bool show = true;
+        int frames = 0;
+        private float lastDisplayUpdate = 0;
+        private bool resetNextFrame = false;
+
+
+        void Awake()
+        {
+            fpsGUIText = GetComponent<UnityEngine.UI.Text>();
```

Comment:
We should probably require this component for the class if we're always expecting to have it.

What about if someone uses `TextMesh` Instead?

Reply:
I suppose they would need to modify the script slightly.  This is intended to be used mostly in the prefab is comes in.  I have added the require component, however.